src/components/navigation/navigation.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Navigation from './navigation';
+import { render } from '@testing-library/react';
+
+describe('<Navigation />', () => {
+  it('should render', () => {
```

Comment:
This test does a bit more then just render, so I would split it. 

First test: just render ;-)
Second test: should contain a title